src/components/Create/BudgetForm.tsx

```diff
@@ -50,47 +52,52 @@ export default function BudgetForm({
 
   return (
     <Space direction="vertical" size="large" style={{ width: '100%' }}>
-      <h1>Funding</h1>
+      <h1>
+        <Trans>Funding</Trans>
+      </h1>
 
       <p>
-        Your project is funded across funding cycles. A funding cycle has a
-        funding target and a duration. Your project's funding cycle
-        configuration will depend on the kind of project you're starting.{' '}
+        <Trans>
+          Your project is funded across funding cycles. A funding cycle has a
+          funding target and a duration. Your project's funding cycle
+          configuration will depend on the kind of project you're starting.
+        </Trans>{' '}
         <a
           href={helpPagePath('protocol/learn/topics/funding-cycle')}
           rel="noopener noreferrer"
           target="_blank"
         >
-          Learn more
+          <Trans>Learn more</Trans>
```

Comment:
**issue**: we should avoid splitting sentences when adding translations. Luckily, linguiJS allows us to wrap this whole block in a `<Trans>`:

```suggestion
<Trans>
          <a
          href={helpPagePath('protocol/learn/topics/funding-cycle')}
          rel="noopener noreferrer"
          target="_blank"
        >
          Learn more
         </a>
         about funding cycles.
         </Trans>
```